File: api/src/main/java/com/calendar/dao/UserDao.java
package com.calendar.dao;

import com.calendar.entity.User;
import org.seasar.doma.Dao;
import org.seasar.doma.Select;
import org.seasar.doma.boot.ConfigAutowireable;

import java.util.List;

@ConfigAutowireable
@Dao
public interface UserDao {

    @Select
    List<User> selectAll();
}

File: src/main/kotlin/com/calendar/calendar/repository/UserDao.kt
package com.calendar.calendar.repository

import com.calendar.calendar.entity.User
import org.seasar.doma.Dao
import org.seasar.doma.Select
import org.seasar.doma.boot.ConfigAutowireable

@ConfigAutowireable
@Dao
interface UserDao {

    @Select
    fun selectAll(): List<User>
}